src/modules/inspect-correlation/inspect-correlation.ts

```diff
@@ -73,6 +73,24 @@ export class InspectCorrelation {
       document.addEventListener('mousemove', mousemoveFunction);
       document.addEventListener('mouseup', mouseUpFunction);
     });
+
+    this.rightPanelResizeDiv.addEventListener('mousedown', (mouseDownEvent: Event) => {
+      const windowEvent: Event = mouseDownEvent || window.event;
+      windowEvent.cancelBubble = true;
+
+      const mousemoveFunction: IEventFunction = (mouseMoveEvent: MouseEvent): void => {
+        this.resizeTokenViewer(mouseMoveEvent);
```

Comment:
Does this have to be public?

```suggestion
        this._resizeTokenViewer(mouseMoveEvent);
```